doc/explanation/releases-snap.md

```diff
@@ -0,0 +1,126 @@
+
+(exp-snap)=
+## The LXD snap
+
+The recommended way to use LXD is through [its snap package](https://snapcraft.io/lxd), if snaps are available for your system. A key benefit of snap packaging is that it includes the required dependencies. This allows LXD to run in a consistent environment on many different Linux distributions. Using the snap also streamlines updates through its channels. 
+
+(exp-snap-channels)=
+### About channels
+
+Each installed LXD snap follows a [channel](https://snapcraft.io/docs/channels). Channels for LXD consist of a track and a risk level, such as `6/stable`. Each channel points to one release at a time, and when a new release is published to a channel, it replaces the previous one. {ref}`Updating the snap <howto-snap-updates>` then updates to that release.
+
+Use only the LTS and feature release channels for LXD. To view all available channels, run:
+
+```bash
+snap info lxd
+```
+
+(exp-snap-tracks)=
+#### Tracks
+
+LXD releases are grouped under [snap tracks](https://snapcraft.io/docs/channels#heading--tracks), such as `6` or `5.21`. For each track, there are three risk levels.
+
+(exp-snap-risk)=
+#### Risk levels
+
+Available [risk levels](https://snapcraft.io/docs/channels#heading--risk-levels) for LXD are `stable`, `candidate`, and `edge`. Use the `stable` risk level to install fully tested releases; this is the only risk level supported under [Ubuntu Pro](https://ubuntu.com/pro). The `candidate` and `edge` levels offer newer but less-tested updates, posing higher risk.
+
+(exp-snap-channels-default)=
+### The default channel
+
+The default channel is recommended for most use cases. It consists of the latest LTS track and the `stable` risk level. The current default channel is {{default_snap_channel}}.
+
+If you {ref}`install the LXD snap <installing-snap-package>` without specifying a channel, the default is used. If you provide the track but not the risk level (for example, `--channel=6`), it defaults to the `stable` risk level.
+
+(exp-snap-channels-lts)=
+### LTS channels
+
+LXD LTS channels use the format _x[.y]_`/[stable|candidate|edge]`, where _x_ and _y_ correspond to the major and minor numbers of {ref}`exp-releases-lts`. Tracks up to `5.21` include both _x_ and _y_, but future LTS tracks will use only _x_.
+
+(exp-snap-channels-feature)=
+### The feature release channel
+
+The LXD feature release channel follows the format _x_`/[stable|candidate|edge]`, with only one active feature track at a time. The current feature track is {{current_feature_track}}.
+
+In the past, each {ref}`LXD feature release <exp-releases-feature>` was published to its own track, such as `5.1` and `5.2`. Starting with LXD `6.1`, releases within the same major version are published to the same track, replacing the previous release. For example, the `6.4` release replaced `6.3` in the `6` track. This simplifies updates, as you don't need to switch channels to access new feature releases within the same major version.   
```

Comment:
Now that all the per minor releases tracks are closed, is it worth documenting this historical fact (the bit about `5.1` and `5.2`)?